app/test/frontend/golden/pkg_show_page.html

```diff
@@ -159,13 +159,13 @@ <h3>3. Import it</h3>
   <td class="documentation">
     <a href="http:&#x2F;&#x2F;www.dartdocs.org&#x2F;documentation&#x2F;foobar_pkg&#x2F;0.1.1&#x2F;"
        title="Go to the documentation of foobar_pkg 0.1.1">
-      <img src="&#x2F;static&#x2F;v2&#x2F;img&#x2F;ic_drive_document_black_24dp.svg" alt="Go to the documentation of foobar_pkg 0.1.1" />
+      <img src="&#x2F;static&#x2F;img&#x2F;ic_drive_document_black_24dp.svg" alt="Go to the documentation of foobar_pkg 0.1.1" />
```

Comment:
Are these links escaped on purpose?

Reply:
No escape needed there, I'll fix it in a follow-up.